### PR TITLE
Rezabekf/readme

### PR DESCRIPTION
**Pull request by rezabekf**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,124 @@
...
+> **Note**  
+You are responsible for the cost of the AWS services used while running this sample deployment. There is no additional cost for using this sample. For full details, see the pricing pages for each AWS service you will be using in this sample. Prices are subject to change.
 
-`$ cp main/.env.example main/.env`
+1. Deploy the latest CloudFormation template by following the link below for your preferred AWS region:
 
-4. Run the deployment script: 
+|Region|Launch Template|
+|------|---------------|
+|**US East (N. Virginia)** (us-east-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-us-east-1/aws-dms-sql-server/v0.1/main.template)|
```
Comment (matteofigus) [inline README.md]:
```suggestion
|**US East (N. Virginia)** (us-east-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-us-east-1/aws-dms-sql-server/latest/main.template)|
```

File: README.md [hunk 2]
```diff
@@ -1,36 +1,124 @@
...
+|**EU (Ireland)** (eu-west-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-eu-west-1/aws-dms-sql-server/v0.1/main.template)|
```
Comment (matteofigus) [inline README.md]:
```suggestion
|**EU (Ireland)** (eu-west-1) | [![Launch the EngagementMeter Stack with CloudFormation](docs/deploy-to-aws.png)](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=aws-dms-sql-server&templateURL=https://s3.amazonaws.com/solution-builders-eu-west-1/aws-dms-sql-server/latest/main.template)|
```

File: README.md (hunk 1 above)
Comment (matteofigus) [reply README.md]: In this way we don't need to update the README every time we do a release

File: README.md [hunk 3]
```diff
@@ -1,36 +1,124 @@
...
+1. You should see a screen titled "*Create Stack*" at the "*Specify template*" step. The fields specifying the CloudFormation 
+template are pre-populated. Click the *Next* button at the bottom of the page.
+1. On the "*Specify stack details*" screen you may customize the following parameters of the CloudFormation stack:
 
+|Parameter label|Default|Description|
+|---------------|-------|-----------|
+|Availability Zones|Requires input|The list of Availability Zones to use for the subnets in the VPCs. *Use two AZs*.|
+|On premise CIDR IP|Requires input|The CIDR Allowed RDP and SQL access to the EC2 and RDS host. CIDR block parameter must be in the form x.x.x.x/0-32.|
+|EC2 instance type|m5.2xlarge|The EC2 instance type for Microsoft SQL server.|
+|Windows server AMI|/aws/service/ami-windows-latest/Windows_Server-2019-English-Full-SQL_2016_SP2_Standard|The latest Windows server 2019 with SQL 2016 Standard AMI.|
```
Comment (matteofigus) [inline README.md]: Can we put a link in the docs here where to find all the AMIs list?

File: README.md [hunk 4]
```diff
@@ -1,36 +1,124 @@
...
+|MSSQL Server version|13|MSSQL Server version. This is used to Change Auth mode from Windows only to SQL and Windows Auth For MSSQL server 2017 use number 14, for MSSQL server 2016 use number 13.|
```
Comment (matteofigus) [inline README.md]: What's the difference between this param and the Database engine version?

File: README.md [hunk 5]
```diff
@@ -1,36 +1,124 @@
...
+|RDS instance type|db.m5.large|Instance class of RDS instance.|
+|Database engine type|sqlserver-se|MS SQL engine type.|
```
Comment (matteofigus) [inline README.md]: Can we put a link to the docs here where to find all the database engine types allowed?

File: README.md [hunk 6]
```diff
@@ -1,36 +1,124 @@
...
+### Remove the application
+
+To remove the application:
+
+1. Open the AWS CloudFormation Console
+1. Click the *aws-dms-sql-server* project, right-click and select "*Delete Stack*"
+1. Your stack will take some time to be deleted. You can track its progress in the "Events" tab.
+1. When it is done, the status will change from DELETE_IN_PROGRESS" to "DELETE_COMPLETE". It will then disappear from the list.
+
+### Making changes to the code and customization
```
Comment (matteofigus) [inline README.md]: Do we need this? I think this message is specific to running front-end and I don't think we have any front-end here

File: README.md (hunk 1 above)
Comment (rezabekf) [reply README.md]: Good catch. Fixed

File: README.md (hunk 2 above)
Comment (rezabekf) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (rezabekf) [reply README.md]: sure...Done

File: README.md (hunk 4 above)
Comment (rezabekf) [reply README.md]: This is to change the way user authenticate to SQL server. I need to modify registry on windows instance to change auth mode. $registryPath = "HKLM:\SOFTWARE\Microsoft\Microsoft SQL Server\MSSQL${MSSQLVersion}.MSSQLSERVER\MSSQLServer"

Comment (rezabekf) [reply README.md]: Database engine version is exclusively used for RDS

File: README.md (hunk 5 above)
Comment (rezabekf) [reply README.md]: good point, i will provide list of the options to chose different engines from that support DMS

File: README.md (hunk 6 above)
Comment (rezabekf) [reply README.md]: I have reword this to delete the stack, as these steps apply to clean up resources.
